### DISCUSSION TITLE
Welcome Appwrite to the Student Developer Pack!

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#144347] [GitHub Education]
![white](https://github.com/user-attachments/assets/3d9dd789-f71e-4f40-bfda-2291f2524666)

We're happy to share that Appwrite is now a part of the Student Developer Pack! Appwrite is an open-source Backend-as-a-Service platform for building web, mobile, native, or backend apps.

If you are a verified student through [GitHub Education](https://education.github.com/students), sign up for a new Appwrite account to gain free access to Appwrite Pro, their plan that’s designed for professional developers or development teams that need to build applications at scale (worth $15/month).

**With a Appwrite Pro, you’ll gain access to everything in the Free plan, plus:**

- 300GB bandwidth
- 150GB storage
- 3.5M executions
- 200K monthly active users
- Organization roles
- Email support
- Daily backups stored for 7 days
- Add-ons
- Unlimited Databases, Buckets, and Functions

Learn more about Appwrite and their GitHub Student Developer Pack offer [here](https://appwrite.io/education).

To get the most out of your student benefits, start by [getting verified](https://education.github.com/students), then explore exclusive student offers from across the industry today.

![github](https://github.com/user-attachments/assets/f0e64f8c-e2c4-42ec-8bfa-975d770845b4)

### Resources to get you started

Explore Appwrite’s CL for GItHub Actions - https://github.com/marketplace/actions/setup-appwrite-cli

Bookmark these!

- [Appwrite's main repo](https://github.com/appwrite/appwrite)
- [Getting Started](https://github.com/appwrite/appwrite#getting-started)
- [Installation](https://github.com/appwrite/appwrite#installation)
- [One-Click Setups](https://github.com/appwrite/appwrite#one-click-setups)
- [Contributing](https://github.com/appwrite/appwrite#contributing)
- [Security](https://github.com/appwrite/appwrite#security)

[Join Appwrite's community](https://appwrite.io/community) of maintainers and contributors and help us make Appwrite better for developers worldwide. Sign up for their Discord to participate in monthly office hours and to learn how you can contribute

**Comment by Ravindulakmina** [comment]
Looks great.

**Comment by W3-Briel** [comment]
woow i need a intitutional email for postulate to github education :(
